user_sync/config.py

```diff
@@ -637,6 +727,91 @@ def describe_unused_values(self):
         if (len(unused_keys) > 0):
             messages.append("Found unused keys: %s in: %s" % (unused_keys, self.get_full_scope()))
         return messages
+
+def load_from_yaml(filename):
```

Comment:
moving these out of the class is not really good unless you're trying to export them from the module separately.  it's better to control their visibility through the class.  why did you do this extract?